maestro-ios-driver/src/main/kotlin/util/LocalSimulatorUtils.kt

```diff
@@ -51,20 +46,13 @@ object LocalSimulatorUtils {
         return jacksonObjectMapper().readValue(json)
     }
 
-    fun awaitLaunch(deviceId: String, timeout: Long= 30000) {
-        MaestroTimer.withTimeout(timeout) {
-            if (list()
-                    .devices
-                    .values
-                    .flatten()
-                    .find { it.udid.equals(deviceId, ignoreCase = true) }
-                    ?.state == "Booted"
-            ) true else null
-        } ?: throw SimctlError("Device $deviceId did not boot in time")
+    fun awaitLaunch(deviceId: String) {
+        val command = listOf("xcrun", "simctl", "bootstatus", deviceId)
+        runCommand(command, waitForCompletion = true)
     }
 
-    fun awaitShutdown(deviceId: String) {
-        MaestroTimer.withTimeout(30000) {
+    fun awaitShutdown(deviceId: String, timeoutMs: Long = 60000) {
```

Comment:
Are we not going to increase this timeout?

Reply:
I increased it just in case, from what I've tested has no functional impact on local cli logic, but might be safer in the cloud